includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -548,6 +548,7 @@ public function submit() {
 				'job_fields'         => $this->get_fields( 'job' ),
 				'company_fields'     => $this->get_fields( 'company' ),
 				'step'               => $this->get_step(),
+				'can_resume_later'   => $this->can_resume_later(),
```

Comment:
Since the word "resume" (meaning "continue") in code is spelled the same as "resumé" (as in, CV), I wonder if it would be better to use a different word to avoid confusion?

Maybe `can_continue_later`?

Reply:
Yup, I was wondering the same but I thought it was fine as a verb. I'll change it though. 71e145dc93ed39f42a3ca2f6b4480448ffe325a7